docs/efun/calls/call_out.md

```diff
@@ -5,23 +5,41 @@ title: calls / call_out
 
 ### NAME
 
-    call_out() - delayed function call in same object
+    call_out - delayed function call in same object
 
 ### SYNOPSIS
 
-    void call_out( string | function fun, int delay, mixed arg );
+    int call_out( string | function fun, int | float delay, mixed arg ... );
 
 ### DESCRIPTION
 
-    Set  up  a  call of function <fun> in this_object(). The call will take
-    place in <delay> seconds, with the argument <arg> provided.  <arg>  can
-    be of any type.
+    Schedule a future call of function <fun> in this_object(). The call will
+    take place in <delay> seconds, with each of the arguments <arg> provided.
+    <arg> can be of any type.
 
-    Please  note  that  you  can't  rely on write() or say() in <fun> since
+    If the gametick in the runtime config is less than 1000, you may specify
+    a <delay> as a float in milliseconds (gametick / 1000) representing a 
+    value that is divisible by the gametick.
```

Comment:
you can still schedule call_out(0.1) with an gametick set to 1000ms, it will simply be executed on next gametick .

Reply:
I've submitted a new commit: https://github.com/fluffos/fluffos/pull/713/commits/94575ed6917b1439bbea29bef914d74bcde6499b